src/index.tsx

```diff
@@ -184,11 +187,15 @@ const OTPInput = ({
     let nextActiveInput = activeInput;
 
     // Get pastedData in an array of max size (num of inputs - current position)
-    const pastedData = event.clipboardData
+    let pastedData = event.clipboardData
       .getData('text/plain')
       .slice(0, numInputs - activeInput)
       .split('');
```

Comment:
I think this should be done once we have removed the space characters from the string. so that we don't skip any characters just because there was a space

Reply:
Yeah makes sense, I updated the code for it @prateek3255